ADD_SAMPLE_INTO_DEV_KIT.md

```diff
@@ -0,0 +1,72 @@
+# How to add samples to sample gallery of Office Add-ins Development Kit
+
+## Configure the sample for the Dev Kit
+
+The Dev Kit sample gallery uses the [samples-config.json](./.config/sample-config.json) file in this repo's `main` branch. It uses the sample configuration ID find the files, downloads a .zip file of the sample project from this repo, and unzips that file in the designated path.
```

Comment:
I'd move this header and text down to the JSON section (replacing that header with this one). Setting up the config JSON is the last step, so it should come near the end of the article.